src/ansys/dpf/post/result_workflows/_utils.py

```diff
@@ -12,6 +13,29 @@ def __call__(self, name: str) -> Operator:
         ...
 
 
+@dataclasses.dataclass
+class AveragingConfig:
+    """Configuration for averaging of results."""
+
+    # List of properties that define a body. The mesh is split by these properties to
+    # get the bodies.
+    body_defining_properties: Optional[list[str]] = None
+    # If True, the results are averaged per body. The bodies are determined
+    # by the body_defining_properties.
+    average_per_body: bool = False
+
+
+default_per_body_averaging_config = AveragingConfig(
+    body_defining_properties=[
```

Comment:
to prevent string duplications, could you please use ansys.dpf.core.common.elemental_properties?